src/modules/75-cf/components/FlagActivation/ServicesList.tsx

```diff
@@ -188,11 +210,28 @@ const ServicesList: React.FC<ServicesListProps> = props => {
   const { showSuccess } = useToaster()
   const { getString } = useStrings()
 
+  const filterOptions = [
```

Comment:
I wonder if you state the type here, can you remove the type assertion on line 225
```suggestion
  const filterOptions: SelectOption[] = [
```

Reply:
Yeah, I suppose `find` can fail to find something. This is fine, so don't worry about it, but an alternative (that I don't know how I feel about) is that you can add a `!` after it to signal to TS that it won't be `undefined`. This is more obvious (IMO), but just an FYI that the other approach exists